app/services/pricing/service.go

```diff
@@ -348,5 +466,10 @@ func initialize(bridgeConfig *config.Bridge, assetsService service.Assets, mirro
 	if err != nil {
 		panic(fmt.Sprintf("Failed to initially fetch USD prices. Error: [%s]", err.Error()))
 	}
+
+	err = instance.FetchAndUpdateNftFeesForApi()
+	if err != nil {
+		panic(fmt.Sprintf("Failed to initially fetch portal/burn fees for NFT assets. Error: [%s]", err.Error()))
+	}
```

Comment:
You call this in `instance.FetchAndUpdateUsdPrices()`, is there any point this to still be called here?